.gitignore

```diff
@@ -0,0 +1,3 @@
+Gemfile.lock
```

Comment:
Also bundler itselfs recommends to check in the file http://bundler.io/v1.7/rationale.html#checking-your-code-into-version-control